### PR TITLE
Кильдишев - Лабораторная работа #1

### PR DESCRIPTION
**Pull request by mkildishev**
@SemenBevzuk @Anna-Mironova

### CODE REVIEW

File: modules/complex-number/test/test_kildishev_maxim_complex_number.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    ComplexNumber z1(z);
+    EXPECT_EQ(z, z1);
+}
+
+TEST(Kildishev_Maxim_ComplexNumberTest, Sum_works) {
+    ComplexNumber z(1.0, 1.0);
+    ComplexNumber z1(2.0, 2.1);
+    ComplexNumber z2 = z + z1;
+    ComplexNumber res(3, 3.1);
+    EXPECT_EQ(res, z2);
```
Comment (kiselik) [inline modules/complex-number/test/test_kildishev_maxim_complex_number.cpp]: I read,that floating point operations can lead to rounding errors. For example, you can get next situation: EXPECT_EQ (3.1, 3.11). the result of this test is false. That's why you need to use special asserts for this numbers: EXPECT_DOUBLE_EQ or EXPECT_FLOAT_EQ

Comment (mkildishev) [inline modules/complex-number/test/test_kildishev_maxim_complex_number.cpp]: Thanks for advice, @kiselik! I will fix it in next commit.

**Comment by SemenBevzuk** [comment]
👍

**Comment by Anna-Mironova** [comment]
👍

**Comment by mkildishev** [comment]
@kirill-korniakov

**Pull request merged**